src/settings-ui/Settings.UI.Library/ViewModels/ColorPickerViewModel.cs

```diff
@@ -216,6 +218,7 @@ private void InitializeColorFormats()
             formatsUnordered.Add(new ColorFormatModel(ciexyzFormatName, "XYZ(59, 35, 98)", visibleFormats.ContainsKey(ciexyzFormatName) && visibleFormats[ciexyzFormatName]));
             formatsUnordered.Add(new ColorFormatModel(vec4FormatName, "(0.94f, 0.41f, 1.00f, 1f)", visibleFormats.ContainsKey(vec4FormatName) && visibleFormats[vec4FormatName]));
             formatsUnordered.Add(new ColorFormatModel(decimalFormatName, "15689983", visibleFormats.ContainsKey(decimalFormatName) && visibleFormats[decimalFormatName]));
+            formatsUnordered.Add(new ColorFormatModel(decimalRGBFormatName, "15689983", visibleFormats.ContainsKey(decimalRGBFormatName) && visibleFormats[decimalRGBFormatName]));
```

Comment:
if one is BGR and other is RGB, one of these two generated values would be wrong i'd imagine.

note: 15689983

Reply:
Fixed to both represent #ef68ff. Added this color to tests as proof.